### PR TITLE
Meta: update roadmap

### PR DESCRIPTION
**Pull request by crodriguezvega**
Update of roadmap for Q1.

### CODE REVIEW

File: meta/ROADMAP.md [hunk 1]
```diff
@@ -1,20 +1,15 @@
...
-- Rough draft of channel upgrade process (potentially connection as well).
-
 ## Q1 - 2022
 
-- Possibly finalize the review and merge of [ICS721](https://github.com/cosmos/ibc/pull/615) (NFT tranfers) specification.
-- Continue review and advisory work on [Informal Systems](https://informal.systems)' [specification proposal for cross-chain validation](https://github.com/cosmos/ibc/pull/563).
-- Begin a re-write of IBC specifications to make them easier to understand to qualified developers trying to implement IBC in other ecosystems. This will most likely be a multi-quarter effort.
\ No newline at end of file
+- Update the Interchain Accounts ([ICS27](https://github.com/cosmos/ibc/blob/master/spec/app/ics-027-interchain-accounts/README.md)) and Relayer Incentivisation ([ICS29](https://github.com/cosmos/ibc/tree/master/spec/app/ics-029-fee-payment)) specifications to align them with the ibc-go implementation.
+- Work on the connection upgradability and channel upgredability specs.
```
Comment (mpoke) [inline meta/ROADMAP.md:12]:
```suggestion
- Work on the connection upgradability and channel upgradability specs.
```

File: meta/ROADMAP.md [hunk 2]
```diff
@@ -1,20 +1,15 @@
...
+- Work on spec for ordered channels that are allowed to timeout.
+- Work on spec to allow timeouts on ordered channels that don't have proof of absence.
```
Comment (mpoke) [inline meta/ROADMAP.md:14]: Is this different from the previous bullet point?

File: meta/ROADMAP.md [hunk 3]
```diff
@@ -1,20 +1,15 @@
...
 
-## Q4 - 2021
-
-- Update the Interchain Accounts ([ICS27](https://github.com/cosmos/ibc/blob/master/spec/app/ics-027-interchain-accounts/README.md)) and Relayer Incentivisation ([ICS29](https://github.com/cosmos/ibc/tree/master/spec/app/ics-029-fee-payment)) specifications to align them with the ibc-go implementation.
-- The function `NegotiateAppVersion` has been [added to the app module interface in ibc-go](https://github.com/cosmos/ibc-go/pull/384) as part of the Interchain Accounts work. This change is not reflected yet in the [ICS05](https://github.com/cosmos/ibc/blob/master/spec/core/ics-005-port-allocation/README.md) specification. An update is needed to describe its semantics beyond the Interchain Accounts use case.
-- Review [IRISnet](https://www.irisnet.org)'s [ICS721](https://github.com/cosmos/ibc/pull/615) specification proposal for NFT transfers.
-- Rough draft of channel upgrade process (potentially connection as well).
-
 ## Q1 - 2022
```
Comment (mpoke) [inline meta/ROADMAP.md:10]: I'd add also ICS28 (i.e., CCV): Finalize a complete draft of ICS28 (Cross-chain validation) that is aligned with the implementation of Interchain Security V1.

**Review by mpoke — APPROVED** [review]
LGTM. See comments below.

**Review by cwgoes — APPROVED** [review]
ACK from me, looks great! looking forward :tada:

File: meta/ROADMAP.md (hunk 3 above)
Comment (crodriguezvega) [reply meta/ROADMAP.md:10]: Indeed; as I was writing the update I thought about it... Thanks for the suggestion. :)

File: meta/ROADMAP.md (hunk 2 above)
Comment (crodriguezvega) [reply meta/ROADMAP.md:14]: Good catch. I think I made a mistake here.

**Comment by mpoke** [comment]
@crodriguezvega Should we merge this?

**Pull request merged**